### DISCUSSION TITLE
Completed plots not passing "plots check"

### DISCUSSION THREAD

**Discussion by sixcomma** [Chia-Network/chia-blockchain#5525] [Q&A]
Hello everyone,
I've been experimenting with different plotting configs in an attempt to optimize my equipment. I think I over subscribed my I/O as the plotting speed really slowed down. One other consequence is I have a number of finished plots that are not showing up as valid when I run plots check. It seems to be random and as the number of concurrent plots dropped the number of invalid plots also decreased. The logs seem okay -- no errors showing up. I'm wondering if it is an issue with writing the final file to disk. Is there a way for me to check this? 

Thanks!

**Comment by uldh** [comment]
"plots check" is a huge problem

**Reply by gromran** [reply]
mostly a layer 8 problem...
